admin/html/views/assistants.php

```diff
@@ -29,7 +29,7 @@ public function pageBtns()
 
     private function assistantsTable()
     {
-        echo '<table class="buddybot-org-assistants-table table table-sm">';
```

Comment:
@shubham9649 why did we removed the table-sm class?

Reply:
we are getting the mobile responsiveness issues then jawed sir remove this to fix it.